src/com/videojs/providers/HTTPVideoProvider.as

```diff
@@ -574,8 +573,6 @@ package com.videojs.providers{
                         _model.broadcastEvent(new VideoPlaybackEvent(VideoPlaybackEvent.ON_STREAM_CLOSE, {info:e.info}));
                         _model.broadcastEventExternally(ExternalEventName.ON_PAUSE);
                         _model.broadcastEventExternally(ExternalEventName.ON_PLAYBACK_COMPLETE);
-                        _startOffset = 0;
-                        _pausedSeekValue = 0;
```

Comment:
https://github.com/videojs/video-js-swf/pull/170/files

They came from this one, which has already been accepted into master. I was just consolidating the two PRs into one for the port to stable.